src/pages/chainlink-automation/tutorials/nft-quickstart.mdx

```diff
@@ -0,0 +1,113 @@
+   />
+
+1. At this stage, you will need to create a VRF subscription and fund it with LINK by visiting the [Chainlink VRF Subscription Manager](https://vrf.chain.link/). Follow the steps outlined in the interface to create a subscription. You will receive a confirmation message upon successful creation.
+
+1. Add funds to your subscription. Copy the resulting subscription ID to your clipboard.
+
+<ClickToZoom
+  src="/images/automation/tutorials/batch-nft3.png"
+  alt="Batch NFT Quickstart Step 3"
+  style="max-width: 70%;"
+/>
+
+<ClickToZoom
+  src="/images/automation/tutorials/batch-nft4.png"
+  alt="Batch NFT Quickstart Step 4"
+  style="max-width: 70%;"
+/>
+
+1. Navigate back to the demo app. Paste the VRF Subscription ID into the **Subscription ID** field and fill out the fields for **Queue Size Trigger** and **Time Trigger (seconds)**.
+
+1. Upon successful creation, you will see a confirmation page. Copy the address and navigate back to the VRF Subscription Manager.
+
+<ClickToZoom
+  src="/images/automation/tutorials/batch-nft5.png"
+  alt="Batch NFT Quickstart Step 5"
+  style="max-width: 70%;"
+/>
+
+1. In the subscription manager, click **Add Consumer** and paste the address in the blank field. Click **Add Consumer** and wait until you receive confirmation. Your consumer should no be added to your subscription. _Note_: you may have to refresh the page to see the consumer.
+
+<ClickToZoom
+  src="/images/automation/tutorials/batch-nft6.png"
+  alt="Batch NFT Quickstart Step 6"
+  style="max-width: 70%;"
+/>
+
+1. The next step is to register a new Upkeep using Chainlink Automation. Visit the [Chainlink Automation App](https://automation.chain.link/) to get started. Click the button **Register New Upkeep** and connect your wallet.
+
+1. You will be directed to a screen where you will choose your Trigger from a dropdown menu. Select **Custom logic**. Paste your address into the _Target contract address_ field. Then, click **Next**.
+
+1. You will be directed to a form to fill out more information on your upkeep. Fill out these details and set the gas limit to 200000 to ensure the Chainlink Automation network can perform the upkeep when the trigger conditions are met. Once all the information is filled in, click **Register Upkeep**. Wait until the upkeep has been registered and click **View Upkeep**. You'll be able to see all the details of your Upkeep.
+
+1. Navigate back to the demo app page. Check off both boxes and click **View Collection**. You screen will look similar to this.
+
+<ClickToZoom
+  src="/images/automation/tutorials/batch-nft7.png"
+  alt="Batch NFT Quickstart Step 7"
+  style="max-width: 70%;"
+/>
+
+1. Mint NFTs until the trigger condition is met. At that point, the demo app will display the message "Pending Batch Reveal".The Chainlink Automation upkeep eill be triggered automatically. This will internally make a call to Chainlink VRF for randomness and assign the random traits to the tokens for the reveal.
```

Comment:
```suggestion
1. Mint NFTs until the trigger condition is met. At that point, the demo app will display the message "Pending Batch Reveal". The Chainlink Automation upkeep will be triggered automatically. This will internally make a call to Chainlink VRF for randomness and assign the random traits to the tokens for the reveal.
```